moto/packages/cfnresponse/cfnresponse.py

```diff
@@ -0,0 +1,124 @@
+"""AWS cfnresponse Lambda script."""
+
+# Modified version of the official AWS cfnresponse-module
```

Comment:
TODO: We can probably use the official AWS version, without modifications?
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-lambda-function-code-cfnresponsemodule.html